style.css

```diff
@@ -3376,8 +3376,12 @@ p > video {
 		width: 71.42857144%;
 	}
 
-	body:not(.search-results) .type-post .entry-content blockquote.alignleft {
+	body:not(.search-results) .type-post .entry-content blockquote.alignleft.farleft {
 		margin-left: -40%;
+	}
+
+	body:not(.search-results) .type-post .entry-content blockquote.alignleft {
```

Comment:
I think we should make only `.farleft` to be wider: `(60% - 1.4736842105em)`, and then we don't need to reset the left margin. We can also remove the edit above.